docs/source/core_concepts/glossary.rst

```diff
@@ -192,6 +182,16 @@ of signatures required to consider a transaction signed.
 The default value is 1.
 Each account can link additional public keys and increase own quorum number.
 
+Role
```

Comment:
Could you please revert reordering, it differs from the pr aim

Reply:
Hmm. It was an intended change. Now the topics are sorted alphabetically if no other more valuable factors involved (e.g. logical composition).